assessment_store/scripts/derive_assessment_values.py

```diff
@@ -0,0 +1,53 @@
+import click
+
+from application_store.db.queries.application.queries import get_application
+from assessment_store.db.models.assessment_record.assessment_records import AssessmentRecord
+from assessment_store.db.queries.assessment_records._helpers import derive_application_values
+
+
+@click.command()
+@click.option(
+    "-a",
+    "--application_id",
+    help="Application ID to derive values",
+    prompt=True,
+)
+# @click.option(
+#     "-c",
+#     "--commit",
+#     is_flag=True,
+#     default=False,
+#     show_default=True,
+#     help="Whether to commit the derived values to assessment_store",
+#     prompt=False,
+# )
+def derive_assessment_values(application_id):
+    from flask import current_app
+
+    from db import db
+
+    current_app.logger.info(
+        "Deriving values for {application_id}",
+        extra=dict(application_id=application_id),
+    )
+
+    application_json = get_application(application_id, include_forms=True, as_json=True)
+    derived_values = derive_application_values(application_json)
+    current_app.logger.info("Derived the following values {derived_values}", extra=dict(derived_values=derived_values))
+
+    if click.confirm("Do you want to commit those values to the database?"):
+        assessment_record: AssessmentRecord = db.session.get(AssessmentRecord, application_id)
+        assessment_record.location_json_blob = derived_values["location_json_blob"]
+        assessment_record.funding_amount_requested = derived_values["funding_amount_requested"]
+        assessment_record.asset_type = derived_values["asset_type"]
+        db.session.commit()
+        current_app.logger.info("Database updated with derived values")
+    else:
+        current_app.logger.info("Database not updated")
+
+
+if __name__ == "__main__":
+    from app import app
+
+    with app.app.app_context():
+        derive_assessment_values()
```

Comment:
Random-ish question - should we make this an invoke command rather than a script? Do we have a rationalise for using one over the other?

Reply:
I guess because I knew how to use click to do the confirmation stuff? No real rationale, yes we should standardise